src/styles/elements.css

```diff
@@ -248,6 +248,16 @@ span > code {
   border-radius: 0.25rem;
 }
 
+h1 > code, 
+h2 > code, 
+h3 > code, 
```

Comment:
QQ, why use `>` instead of just `h1 code`? Is there a situation where we'd want `>`?